lib/commands/find.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import { errors } from 'appium-base-driver';
+import { util } from 'appium-support';
+
+
+const commands = {};
+
+commands.findElOrEls = async function findElOrEls (strategy, selector, mult, context) {
```

Comment:
seems like we could have a generic `findElOrElsByProxy` method since we do this in many drivers.?

Reply:
yes, this improvement could be done in a separate PR